dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'Magento_AdobeStockImageAdminUi/js/components/grid/column/overlay'
+], function ($, Overlay) {
+    'use strict';
+
+    describe('Magento_AdobeStockImageAdminUi/js/components/grid/column/overlay', function () {
+
+        var overlayObj;
+
+        beforeEach(function () {
+            overlayObj = new Overlay({
+                provider: 'providerName',
+                licensed: {}
+            });
+        });
+
+        describe('"initObservable" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('initObservable')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.initObservable;
+
+                expect(type).toEqual('function');
+            });
+
+            it('licensed will be defined', function () {
+                expect(overlayObj.licensed).toBeDefined();
+            });
+        });
+
+        describe('"updateLicensed" method', function () {
+
+            var handler = {
+                success: function(data) {
+                    return [1,2]
+                },
+                fail: function(data) {
+                    return 'Error Message';
+                }
+            };
+            var testObj = {
```

Comment:
Hmm, why create a new Ajax-invoking method here housed in a `testObj`? I think we could remove this object and instead, in the `updateLicensed` test below, invoke `Overlay.updateLicensed` directly.